shared/src/Unison/Node/Builtin.hs

```diff
@@ -105,6 +105,11 @@ makeBuiltins whnf =
            op [a] = pure $ Term.remote (Remote.Step (Remote.Local (Remote.Pure a)))
            op _ = fail "unpossible"
        in (r, Just (I.Primop 1 op), remoteSignatureOf "Remote.pure", prefix "pure")
+     , let r = R.Builtin "Remote.map"
```

Comment:
I’d have thought map could be written with unison code if desired.  Maybe I’m forgetting something though.